app/models/notification.rb

```diff
@@ -17,13 +17,22 @@ class Notification < ApplicationRecord
   scope :read,    -> { where.not(read_at: nil) }
   scope :unread,  -> { where(read_at: nil) }
 
+  # All unread notifications within a given span of time
+  scope :current, -> (time_ago=1.day) {
```

Comment:
```suggestion
  scope :since, -> (time_ago=1.day) {
```
`current` doesn't help me understand what current notifications would be for. Since, with a time makes more sense to read out loud. `Notification.since 1.day`